test/framework/src/main/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -1731,6 +1731,20 @@ public static boolean indexExists(String index, Client client) {
         return getIndexResponse.getIndices().length > 0;
     }
 
+    public static void ensureIndexExists(String index) {
```

Comment:
Naming nit, maybe `awaitIndexExists` to show that it will wait? Otherwise this reads to me as something that will immediately fail if the index doesn't exist.

Reply:
Yep renamed as suggested. Thanks!